src/main/java/com/softserveinc/ita/rozetka/page_objects/OrderPage.java

```diff
@@ -1,37 +1,50 @@
 package com.softserveinc.ita.rozetka.page_objects;
 
 import static com.codeborne.selenide.Selenide.$x;
-import static java.lang.String.format;
 
 public class OrderPage {
 
-    public OrderPage setSurname(String surname) {
-        $x("//div[@class='form__row js-surname']/input").setValue(surname);
-        return this;
-    }
-
-    public OrderPage setName(String name) {
-        $x("//div[@class='form__row js-name']/input").setValue(name);
-        return this;
-    }
-
     public BasketPage startEditingProductsInBasket() {
         $x("//*[@class='button button_with_icon button_type_link checkout-product__edit-button']").click();
         return new BasketPage();
     }
 
-    public String getHeaderText() {
+    public String getPageTitle() {
         return $x("//h1[@class='checkout-heading']").text();
     }
 
-    public OrderPage setPhone(String phone) {
-        $x("//input[@type='tel']").setValue(phone);
-        return this;
+    public boolean isConfirmOrderButtonDisplayed() {
+        return $x("//div[@class='checkout-total__buttons']").isDisplayed();
+    }
+
+    public static String getSurname() {
+        return $x("//div[@class='form__row js-surname']/input").getValue();
+    }
+
+    public static String getName() {
+        return $x("//div[@class='form__row js-name']/input").getValue();
+    }
+
+    public static String getPhone() {
+        return $x("//input[@type='tel']")
+                .getValue()
+                .replaceAll(" ", "");
+    }
+
+    public static String getCity() {
+        return $x("//div[@class='form__row js-city']//input[@name='search']")
+                .getValue()
+                .toLowerCase();
     }
 
-    public OrderPage setCity(String city) {
-        $x("//div[@class='form__row js-city']//input[@name='search']").setValue(city).click();
-        $x(format("(//li[contains(text(),'%s')])[1]", city)).click();
+    public OrderPage setShippingAddress(ShippingAddress shippingAddress) {
+        $x("//div[@class='form__row js-surname']/input").setValue(shippingAddress.getSurname());
+        $x("//div[@class='form__row js-name']/input").setValue(shippingAddress.getName());
+        $x("//input[@type='tel']").setValue(shippingAddress.getPhone());
+        $x("//div[@class='form__row js-city']//input[@name='search']")
+                .setValue(shippingAddress.getCity())
+                .click();
+        $x("//div[@class='form__row js-city']//li[1]").click();
```

Comment:
you should have both setShippingAddress() and setSurname(), setName() etc methods and just those methods in setShippingAddress()